Library/Homebrew/test/dev-cmd/pr-pull_spec.rb

```diff
@@ -62,6 +85,23 @@ class Foo < Formula
           expect(path.git_commit_message).to include("Co-authored-by: #{secondary_author}")
         end
       end
+
+      it "squashes a cask correctly" do
```

Comment:
I think it'd be worth combining these tests if possible. Integration tests are slow af.